### DISCUSSION TITLE
[Script request] Invoiceninja

### DISCUSSION THREAD

**Discussion by ismaelster** [community-scripts/ProxmoxVE#952] [Request script]
### Application Name

Invoiceninja

### Website

https://www.invoiceninja.org/

### Description

Application Name
invoiceninja

Website
https://www.invoiceninja.org/

Description
Self-Hosting Invoice Ninja
Invoices, Expenses and Tasks built with Laravel, Flutter and React

I have found invoiceninja inside turnkey but is it posible to make a lxc for invoicninja?

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
Duplicate.
